packages/domain/src/entities/Credentials.ts

```diff
@@ -2,9 +2,7 @@ import { EvmAddress } from '@lens-protocol/shared-kernel';
 
 import { ProfileId } from './Profile';
 
-export interface ICredentials {
+export type Credentials = {
   readonly address: EvmAddress;
   readonly profileId?: ProfileId;
-
-  isExpired(): boolean;
```

Comment:
Never used in domain layer. It does no longer justify the use of an interface, hence the refactoring into `Credentials` and correlated changes.